streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -432,6 +437,10 @@ public GroupAssignment assign(final Cluster metadata, final GroupSubscription gr
 
             // compute the assignment of tasks to threads within each client and build the final group assignment
 
+            getApplicationState(
```

Comment:
I know we're just throwing away the return value for now but I'd still do this:
```suggestion
            final ApplicationState applicationState = getApplicationState(
```
Otherwise it kind of seems like this method is supposed to be mutating the input parameters (many of the StreamsPartitionAssignor methods work this way so it's good to distinguish when we're just building something vs operating on the passed in structures)

Reply:
SpotBugs was complaining about unused variables when I had that, so I will simply leave the invocation of that new method out entirely.

Reply:
ah, I see. Well there's no concern about forgetting to put this back in since we'll eventually need the result lol. All good 👍